### DISCUSSION TITLE
Saying everything twice

### DISCUSSION THREAD

**Discussion by Belliqueu** [EDCD/EDDI#2493] [EDDI Standalone Q&A]
EDDI is saying everything twice. Once in its own "voice" and once using my Windows default voice at the same time. I thought it was an echo at first, but my Windows default is Sean which has a different accent.

**Comment by Tkael** [comment]
Only one voice should be speaking at a time, though we do actually add a slight echo effect to the voice in addition to some other effects. What makes you certain that it is a distinct voice and not a voice effect?
- On the Text-to-Speech tab, could you dial the `Effect` slider down to zero and see if that change has an effect on what you are observing? Do you observe these double voices when you use the `Test voice` button or only when playing?
- Is your ship damaged? If so there's also an option to add additional distortion effects to your ship when it is damaged. Perhaps that could be what you are observing?

Could you share a screenshot of your Text-to-Speech settings? (and possibly a video recording to demonstrate what you are observing?)
Are you using VoiceAttack?
